notification/tasks.py

```diff
@@ -0,0 +1,62 @@
+from textwrap import dedent
+
+from django.utils import timezone
+
+from celery import task
+from notifications_plugin.models import NotificationMessage, Notification
+
+from yaksh.models import Course, Quiz, QuestionPaper, AnswerPaper
+
+
+@task(name='course_deadline_task')
+def course_deadline_task():
+    courses = Course.objects.filter(active=True, is_trial=False)
+    for course in courses:
+        if course.is_active_enrollment():
+            message = dedent("""
+                The deadline for the course {0} is {1}, please complete
+                the course if not completed before the deadline.
+                """.format(course.name, course.end_enroll_time)
+            )
+            students = course.students.all()
+            creator = course.creator
+            if students:
+                students_id = students.values_list('id', flat=True)
```

Comment:
You can directly do values_list as `course.students.values_list('id', flat=True)`